npm run visual-test - does nothing

It just shows the files in folders

Hi good day, that being the case, how do we load http://localhost:8080/public/visual#/ in the browser after running npm run visual-start ?

Also, I have the following error after running npm run visual-start

<code>
npm ERR! react-router-bootstrap@0.16.0 visual-test: `webpack-dev-server --config webpack.test.config`
npm ERR! Exit status 8
npm ERR!
npm ERR! Failed at the react-router-bootstrap@0.16.0 visual-test script.
npm ERR! This is most likely a problem with the react-router-bootstrap package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     webpack-dev-server --config webpack.test.config
npm ERR! You can get their info via:
npm ERR!     npm owner ls react-router-bootstrap
npm ERR! There is likely additional logging output above.
npm ERR! System Darwin 14.3.0
npm ERR! command "node" "/Users/myusernae/local/bin/npm" "run" "visual-test"
npm ERR! cwd /Users/myusername/learn_react/tobias/testing/react-router-bootstrap
npm ERR! node -v v0.10.24
npm ERR! npm -v 1.3.23
npm ERR! code ELIFECYCLE
npm ERR!
npm ERR! Additional logging details can be found in:
npm ERR!     /Users/myusername/learn_react/testing/react-router-bootstrap/npm-debug.log
npm ERR! not ok code 0
</code>

--- a/package.json
+++ b/package.json
@@ -8,7 +8,7 @@
     "build": "babel src --out-dir=lib && webpack && COMPRESS=1 webpack",
     "test": "npm run lint && karma start --single-run",
     "tdd": "karma start",
-    "visual-test": "webpack-dev-server --config webpack.test.config",
+    "visual-test": "webpack-dev-server --config webpack.test.config.babel.js",
     "lint": "eslint ./"
   },
   "repository": {

